components/bl-draggable-list-component/src/draggable-list/add-item.js

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useState } from 'react';
+
+import { AddSVG, DoneSVG } from '../svg/index';
+import { getContextForAddEvent } from '../utils/context';
+
+export function AddItem({ isAdded, toggleIsAdded, component, eventHandlers }) {
+  const { itemsArray } = component;
+  const { onAdd } = eventHandlers;
+  const [newItem, setNewItem] = useState({ label: '', value: '' });
+
+  const handleChange = useCallback(event => {
+    setNewItem({ ...newItem, [event.target.name]: event.target.value });
+  }, [event, newItem]);
+
+  const handleDone = useCallback(() => {
+    if (newItem.label.trim() && newItem.value.trim()) {
+      const contextForAddEvent = getContextForAddEvent(itemsArray, newItem);
+      onAdd(contextForAddEvent);
```

Comment:
```
onAdd({
    currentArray: itemsArray.concat(newItem),
    addedItem   : newItem,
})
```

I'm not sure if there are needed getContextForAddEvent functions, as it not contain a lot of logic